lang/nl/texts/data.en.html

```diff
@@ -87,13 +87,13 @@ <h2>Android and iPhone mobile app</h2>
 <h2>Wrappers and SDKs</h2>
 <p>We probably have a wrapper for your favorite programming language. If we do, you can use it and improve it. If we don't, you can help create it.</p>
 
-<h2>Mailing list for data, API and exports</h2>
+<h2>Mailinglijst voor gegevens, API en export</h2>
 
-<p>You can also join the mailing list to discuss the API and exports, tell us about your re-uses, and be notified when improvements or changes are made to the API. Send an empty e-mail to
-<a href="mailto:api-subscribe@openfoodfacts.org">api-subscribe@openfoodfacts.org</a> to subscribe.</p>
+<p>U kunt ook deelnemen aan de mailinglijst om de API en export te bespreken, ons te vertellen over uw hergebruik en u kunt op de hoogte worden gesteld wanneer er verbeteringen of wijzigingen worden aangebracht in de API. Stuur een lege e-mail naar
+<a href="mailto:api-subscribe@openfoodfacts.org">api-abonnees@openfoodfacts.org</a> om te abonneren.</p>
```

Comment:
That mail address probably does not exist? The `mailto` link isn't affected, but copying this address would not work.
```suggestion
<a href="mailto:api-subscribe@openfoodfacts.org">api-subscribe@openfoodfacts.org</a> om te abonneren.</p>
```

Reply:
Right, we need to change that to tell them to come on Slack #api instead.